src/daemon/rpmostree-sysroot-upgrader.cxx

```diff
@@ -419,6 +419,16 @@ rpmostree_sysroot_upgrader_pull_base (RpmOstreeSysrootUpgrader  *self,
 
   switch (refspec_type)
     {
+    case RPMOSTREE_REFSPEC_TYPE_CONTAINER:
+      {
+        if (override_commit)
+          return glnx_throw (error, "Specifying commit overrides for container-image-reference type refspecs is not supported");
+
+        auto commit = rpmostreecxx::import_container(*self->sysroot, std::string(refspec));
```

Comment:
Can we still print some sort of progress icon like the other commands? IIUC those make use of libostree's async_progress?

Reply:
Yes, but doing so would need custom code that bridges the progress from the Rust side into `rpmostree-output.h`.